index.bs

```diff
@@ -4557,6 +4557,12 @@ macros:
 The number of channels of the output always equals the number of
```

Comment:
This "always" is no longer right, and so would be best removed.

Perhaps something like:
"For rendering quanta in which some playback occurs, the number of channels of
output equals [...].  For rendering quanta in which no playback occurs, the
output is a single channel of silence."

If you prefer something more similar to the text proposed here, then maybe
... "channel of silence for rendering quanta after and not including the
quantum during which the earliest of the following occurs: the end of the" ...

Reply:
Thank you, @hoch.

```suggestion
The number of channels of the output equals the number of
```

The remaining suggestions here were too precise for what rtoy wanted.